companies/heise-de.json

```diff
@@ -4,14 +4,31 @@
         "de"
     ],
     "name": "Heise Medien GmbH & Co. KG",
+    "runs": [
+        "heise Academy",
+        "Heise Business Services (HBS)",
+        "heise-events.de",
+        "jobs.heise.de",
+        "Mobile Apps",
+        "social.heise.de",
+        "shop.heise.de",
+        "techstage.de",
+        "telepolis.de"
+    ],
     "address": "Karl-Wiechert-Allee 10\n30625 Hannover\nDeutschland",
-    "phone": "+49 511 53520",
-    "fax": "+49 511 5352129",
     "email": "datenschutz@heise.de",
     "web": "https://www.heise.de",
     "sources": [
         "https://www.heise.de/impressum.html",
-        "https://www.heise.de/Datenschutzerklaerung-der-Heise-Medien-GmbH-Co-KG-4860.html"
+        "https://www.heise.de/Datenschutzerklaerung-der-Heise-Medien-GmbH-Co-KG-4860.html",
+        "https://www.heise.de/Individuelle-Datenschutzinformationen-fuer-heise-Academy-und-heise-Academy-Events-4058183.html",
+        "https://www.heise.de/Individuelle-Datenschutzinformationen-fuer-heise-https://www.heise.de/Individuelle-Datenschutzinformationen-fuer-Heise-Business-Services-7099857.html",
```

Comment:
this link is broken!

Reply:
https://www.heise.de/Datenschutzerklaerung-der-Heise-Medien-GmbH-Co-KG-4860.html is fine for me.

![Screenshot_20231017_003403_Firefox Nightly](https://github.com/datenanfragen/data/assets/43509007/f4d97be6-d8e5-4868-8863-3f5cee8d798b)